Automattic/wp-calypso
# Combine Activity Logs

#### Changes proposed in this Pull Request

* Promoted site offset context, activity card, and activity card list to regular calypso components
* move backups/activity-log to activity/activity-log-v2
* control access to which activity log is used via new feature flag, `activity-log/v2`, and activate for all jetpack cloud environments
* refine activity log v2 for WordPress.com
* standardize route for activity log to `/activity-log` across WordPress.com and Jetpack.com

<img width="1899" alt="Screen Shot 2020-06-16 at 10 26 36 AM" src="https://user-images.githubusercontent.com/2810519/84807290-e097a080-afbb-11ea-8a92-d53e18b9717e.png">

#### Testing instructions

1. boot Calypso then navigate to `/activity-log` for a jetpack site
2. verify the section matches production WordPress.com
3. append `?flags=activity-log/v2` to the url
4. verify the section changes to the v2 activity log and works as expected
5. boot Jetpack Cloud , then navigate to `/activity-log`
6. verify the section matches production Jetpack.com
7. append `?flags=jetpack/features-section` to the url
8. verify the activity log section of the new menu works ( highlights when expected, leads to the correct section )
